ai_ref_kits/multimodal_ai_visual_generator/README.md

```diff
@@ -66,47 +65,57 @@ git clone https://github.com/openvinotoolkit/openvino_build_deploy.git
 The above will clone the repository into a directory named "openvino_build_deploy" in the current directory. Then, navigate into the directory using the following command:
 
 ```shell
-cd openvino_build_deploy/ai_ref_kits/multimodal_ai_visual_generator
+cd openvino_build_deploy/ai_ref_kits/ai_adventure_experience
 ```
 
-Next, you’ll download and optimize the required models. This will involve the creation of a temporary virtual environment and the running of a download script. Your requirements.txt file will depend on the Python version you're using (3.11 or 3.12).
+Next, you’ll download and optimize the required models. This will involve the creation of a temporary virtual environment and the running of a download script. 
 
-- nanoLLaVA (multimodal): Image recognition/captioning from webcam 
 - Whisper: Speech recognition
-- Llama3-8b-instruct: Prompt refinement
+- Llama3-8b-instruct: Intelligent LLM helper
 - Latent Consistency Models: Image generation
-  
-**Note:** If you would like to run Latent Consistency Models on the NPU, as shown in the demo above, please follow the following steps: Download the model from this location "https://huggingface.co/Intel/sd-1.5-lcm-openvino" and compile it via the steps located at https://github.com/intel/openvino-ai-plugins-gimp/blob/v2.99-R3-staging/model_setup.py. 
-
-- AI Super Resolution: Increase the resolution of the generated image
+- Super Resolution: Increase the resolution of the generated image
 - Depth Anything v2: Create 3d parallax animations
-    
+
+Linux:
 ```shell
+cd models
 python3 -m venv model_installation_venv
 source model_installation_venv/bin/activate
-pip install -r requirements.txt
+pip install -r model_requirements.txt
```

Comment:
Can we merge these requirements into the main requirements file? It'd be much easier to deal with the CI, then.